test/Microsoft.Framework.DependencyInjection.Tests/ActivatorUtilitiesTests.cs

```diff
@@ -79,12 +79,24 @@ public void TypeActivatorWorksWithStaticCtor(CreateInstanceFunc createFunc)
             Assert.NotNull(anotherClass);
         }
 
-        [Theory]
-        [MemberData(nameof(CreateInstanceFuncs))]
-        public void TypeActivatorWorksWithCtorWithOptionalArgs(CreateInstanceFunc createFunc)
```

Comment:
Yes since I changed GetService to GetRequiredService, if type is not provided, it throws an exception saying type is not registered. Earlier, it would just return null.